<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graduation</class>
 <widget class="QMainWindow" name="graduation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>840</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>자세교정유도프로그램</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Korean" country="SouthKorea"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	Background-color: #FFF;

	border-radius: 12.5px;
	border-style: solid;
	border-color: #bbbbbb;
	border-width: 1px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: none;
	image: url(image/btn_connect.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="record_save_btn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: none;
	image: url(image/btn_record_gray.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="play_stop_btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: none;
	image: url(image/btn_play_gray.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border-radius: 0px;
	border-style: solid;
	border-color: #d3d3d3;
	border-width: 2.5px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="turtleNeck_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
	border-style: none;
	color: #5c5d5d;
}

QCheckBox::indicator:unchecked {
/*	background-color: #bbbbbb; */
	image: url(image/unchecked.png);
}

QCheckBox::indicator:checked {
/*	background-color: #1b95d4; */
	image: url(image/checked(blue mark).png);
}
</string>
      </property>
      <property name="text">
       <string>Forward Head Posture Detection</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="twistedBack_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
	border-style: none;
	color: #5c5d5d;
}

QCheckBox::indicator:unchecked {
/*	background-color: #bbbbbb; */
	image: url(image/unchecked.png);
}

QCheckBox::indicator:checked {
/*	background-color: #1b95d4; */
	image: url(image/checked(blue mark).png);
}
</string>
      </property>
      <property name="text">
       <string>Slant Posture Detection</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="handOnFace_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
	border-style: none;
	color: #5c5d5d;
}

QCheckBox::indicator:unchecked {
/*	background-color: #bbbbbb; */
	image: url(image/unchecked.png);
}

QCheckBox::indicator:checked {
/*	background-color: #1b95d4; */
	image: url(image/checked(blue mark).png);
}
</string>
      </property>
      <property name="text">
       <string>Hand On Face Detection</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="alert_file_setting_groupbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>High Tower Text</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QGroupBox {
	Background-color: #FFF;
	border-radius: 12.5px;
	border-style: solid;
	border-color: #bbbbbb;
	border-width: 1px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color:#1b95d4;
	background-color:#ffffff;
	border-style:none;
}</string>
     </property>
     <property name="text">
      <string>Alert files</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>172</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color:#5c5d5d;
	border-style:none;
	background-color: #ffffff
}</string>
     </property>
     <property name="text">
      <string>Forward Head Posture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="turtle_neck_alert_file_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: #FFF;

border-radius: 0px;
border-style: solid;
border-color: #bbbbbb;
border-width: 1px;

color:#5c5d5d;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>105</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color:#5c5d5d;
	border-style:none;
	background-color: #ffffff
}</string>
     </property>
     <property name="text">
      <string>Slant Posture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="twisted_back_alert_file_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: #FFF;

border-radius: 0px;
border-style: solid;
border-color: #bbbbbb;
border-width: 1px;

color:#5c5d5d;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color:#5c5d5d;
	border-style:none;
	background-color: #ffffff
}</string>
     </property>
     <property name="text">
      <string>Hand On Face</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="hand_on_face_alert_file_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: #FFF;

border-radius: 0px;
border-style: solid;
border-color: #bbbbbb;
border-width: 1px;

color:#5c5d5d;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="turtle_neck_file_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>9</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: none;
	image: url(image/set.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="twisted_back_file_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
         QPushButton {
         border-style: none;
         image: url(image/set.png);
         }
       </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="hand_on_face_file_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
         QPushButton {
        	border-style: none;
         	image: url(image/set.png);
         }
       </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="forward_head_posture_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #FFF;

border-radius: 12.5px;
border-style: solid;
border-color: #bbbbbb;
border-width: 1px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="forward_head_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#1b95d4;
border-style:none;</string>
     </property>
     <property name="text">
      <string>Forward Head Posture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="rgb_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: #000;
border-radius: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="turtle_neck_progress_bar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>349</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar { 
	Background-color: #e6e6e5;
	border-radius: 0px;
}

QProgressBar::chunk { 
	background: #1b95d4; 
}</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLabel" name="forward_head_posture_progressbar_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>349</y>
       <width>77</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	Background: none;
	border-style:none;
	color: #fff;
}
</string>
     </property>
     <property name="text">
      <string>0 grade</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eff2f4;
border-style: none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="slant_posture_groupbox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #FFF;

border-radius: 12.5px;
border-style: solid;
border-color: #bbbbbb;
border-width: 1px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="slant_posture_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#1b95d4;
border-style:none;</string>
     </property>
     <property name="text">
      <string>Slant Posture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ir_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: #000;
border-radius: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="twisted_back_progress_bar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>349</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar { 
	Background-color: #e6e6e5;
	border-radius: 0px;
}

QProgressBar::chunk { 
	background: #1b95d4; 
}</string>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="slant_posture_progressbar_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>349</y>
       <width>77</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	Background: none;
	border-style:none;
	color: #fff;
}
</string>
     </property>
     <property name="text">
      <string>0.0 / 10.0</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eff2f4;
border-style: none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #FFF;

border-radius: 12.5px;
border-style: solid;
border-color: #bbbbbb;
border-width: 1px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="hand_on_face_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#1b95d4;
border-style:none;</string>
     </property>
     <property name="text">
      <string>Hand On Face</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="depth_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: #000;
border-radius: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hand_on_face_count">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>349</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: none;
color: #5c5d5d;</string>
     </property>
     <property name="text">
      <string>Count : 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>12</number>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eff2f4;
border-style: none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>751</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QGroupBox {
	Background-color: #ffffFF;
	border-radius: 12.5px;
	border-style: solid;
	border-color: #bbbbbb;
	border-width: 1px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>731</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="chart_layout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="graduation.qrc"/>
 </resources>
 <connections/>
</ui>
